docs/antrea-agent-simulator.md

```diff
@@ -30,24 +30,18 @@ after deploying it.
 To prevent Pods from being scheduled on the simulated Node(s), you can use the
 following taint.
 
-  ```bash
+```bash
 kubectl taint -l 'antrea/instance=simulator' node mocknode=true:NoExecute
-  ```
+```
 
 ## Create secret for kubemark
 
-  ```bash
-kubectl create secret generic kubeconfig --type=Opaque --namespace=kube-system --from-file=<path to kubeconfig file>
-  ```
+```bash
+kubectl create secret generic kubeconfig --type=Opaque --namespace=kube-system --from-file=admin.conf=<path to kubeconfig file>
+```
 
 ## Apply the yaml file
 
-  ```bash
+```bash
 kubectl apply -f build/yamls/antrea-scale.yml
-  ```
-
-check the simulated Node:
-
-  ```bash
-kubectl get nodes -l 'antrea/instance=simulator'
```

Comment:
I don't see any hollow Nodes, only simulator Pods via kubectl after deploying antrea-scale.yml, so removed this part.

Reply:
On my testbed I can see simulator nodes like: `antrea-agent-simulator-x` when running this command, could you explain more about the issue?

Reply:
hmm, I was using kind cluster, did you try it on Kind? not sure if it's kind specific issue or K8s version related. Which K8s version are you using?

Reply:
@XinShuYang I have reverted this part, thanks.